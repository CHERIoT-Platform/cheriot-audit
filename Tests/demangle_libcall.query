# Check that we can demangle a library export correctly.
--board inputs/sail.json -j inputs/test-suite.json -q 'export_entry_demangle("token_library", input.compartments.token_library.exports[0].export_symbol)'

