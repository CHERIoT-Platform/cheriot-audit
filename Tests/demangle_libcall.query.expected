{"expressions":["token_obj_unseal(SKeyStruct*, SObjStruct*)"]}
